backend/lib/components/Geo/gmaps/index.js

```diff
@@ -50,8 +66,85 @@ const getLocationByLatLng = async (lat, lng) => {
   };
 };
 
+const getNearbyPlacesByType = async (lat, lng, type) => {
+  const location = { lat, lng };
+  const radius = FACILITY_RADIUS_BY_TYPE[type];
+  const { data } = await client.placesNearby({
+    params: {
+      key: geo.googleMapsApiKey,
+      location,
+      radius,
+      type,
+    },
+  });
+  return data.results.length > 0
+    ? data.results.slice(0, MAX_FACILITIES).map((place) => ({ ...place, type }))
+    : data.results;
+};
+
+const getPlaceDetails = async (place) => {
+  const { data } = await client.placeDetails({
+    params: {
+      fields: FACILITY_DETAILS_FIELDS,
+      key: geo.googleMapsApiKey,
+      place_id: place.place_id,
+    },
+  });
+  return {
+    adr_address: data.result.adr_address,
+    business_status: data.result.business_status,
+    formatted_address: data.result.formatted_address,
+    formatted_phone_number: data.result.formatted_phone_number,
+    geometry_location: data.result.geometry.location,
+    international_phone_number: data.result.international_phone_number,
+    name: place.name,
+    open_now:
+      data.result.opening_hours === undefined || data.result.opening_hours.open_now === undefined
+        ? ""
+        : JSON.stringify(data.result.opening_hours.open_now),
```

Comment:
maybe leave to front-end to handle the formatting?

Reply:
Thanks, ddveloper.  Would like to leave this if it's ok, since it's easier to not have to have the FE parse this out.